src/cmds/scripts/pbs_schema_upgrade

```diff
@@ -217,6 +217,13 @@ upgrade_pbs_schema_from_v1_3_0() {
 
 		ALTER INDEX pbs.job_src_idx RENAME TO job_scr_idx;
 		ALTER TABLE pbs.server ALTER COLUMN sv_jobidnumber TYPE BIGINT;
+		ALTER TABLE pbs.scheduler DROP COLUMN sched_sv_name;
+		ALTER TABLE pbs.queue DROP COLUMN qu_sv_name;
+		ALTER TABLE pbs.resv DROP COLUMN ri_sv_name;
+		ALTER TABLE pbs.job DROP COLUMN ji_sv_name;
+		ALTER TABLE pbs.server
+		DROP COLUMN sv_name,
```

Comment:
It is better to add the keyword 'CASCADE' while dropping the column sv_name. It is because this column has foreign key references from other tables.

Reply:
No all the foreign key constraints should have gone - I see now that the sql file has errors and actually has a reference to pbs_server(sv_name) which does not exist